permission_handler_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.0-nullsafety.0
```

Comment:
Please remove the trailing `.0` as we are following the pattern of not adding a `.0` for the first `nullsafety` release.
```suggestion
## 3.0.0-nullsafety
```